app/models/openscap_result.rb

```diff
@@ -0,0 +1,77 @@
+class OpenscapResult < ApplicationRecord
+  belongs_to :container_image
+  has_one    :binary_blob,           :dependent => :destroy, :autosave => true, :as => :resource
+  has_many   :openscap_rule_results, :dependent => :destroy, :autosave => true
+
+  def self.openscap_available?
+    # needed only for travis
+    require 'openscap'
+    require 'openscap/ds/arf'
+    require 'openscap/xccdf/benchmark'
+    true
+  rescue LoadError
+    false
+  end
+
+  def html
+    raw = binary_blob.try(:binary)
+    return unless raw
```

Comment:
Is it safe for this method to return `nil`?

It looks like this is [sent to `send_data` above](https://github.com/ManageIQ/manageiq/pull/6839/files#diff-12a48123abd31efa3db8f18294aeec46R20).  Wondering if that's fine, or if the caller or this method should check for `nil` and send back an appropriate response.